routes/channels.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Illuminate\Support\Facades\Broadcast;
+
+/*
+|--------------------------------------------------------------------------
+| Broadcast Channels
+|--------------------------------------------------------------------------
+|
+| Here you may register all of the event broadcasting channels that your
+| application supports. The given channel authorization callbacks are
+| used to check if an authenticated user can listen to the channel.
+|
+*/
+
+Broadcast::channel('App.User.{id}', function ($user, $id) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Opening parenthesis of a multi-line function call must be the last content on the line](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)